tools/checked-c-convert/ConstraintBuilder.cpp

```diff
@@ -178,6 +177,9 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
   // In any of these cases, due to conditional expressions, the number of
   // variables on the RHS could be 0 or more. We just do the same rule
   // for each pair of q_i to q_j \forall j in variables_on_rhs.
+  //
+  // V is the set of constraint variables on the left hand side that we are
```

Comment:
Could you amplify this comment some more?   It's the set of constraint variables for the LHS.    The LHS will have a pointer type and the constraint variables represent the constraints for that type.